src/avx512bw/avx512bw-from-valid-utf8.inl.cpp

```diff
@@ -0,0 +1,345 @@
+// file included directly
+
+// File contains conversion procedure from VALID UTF-8 strings.
+
+/*
+    expand_bytes takes 16 + 4 bytes (of a UTF-8 string)
```

Comment:
Right. So you have an input 20 bytes. You have 16 '4-grams' (a technical term) and 4 * 16 = 64 bytes. That makes sense.

Reply:
Hmm, I thought "gram" is more accurate in terms of text. Here we chopped the text representation. I'll update the comment if you think the "gram" is more accurate.

Reply:
No, no, my comments are not requests for change. I am just sharing my thoughts as I read your code. I try to be specific and clear when I want people to change their code.